pkg/lib/configreader/errors.go

```diff
@@ -126,6 +128,13 @@ func ErrorAlphaNumericDashDotUnderscore(provided string) error {
 	})
 }
 
+func ErrorInvalidAWSTag(provided string) error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrInvalidAWSTag,
+		Message: fmt.Sprintf("%s must contain only letters, numbers, spaces, and the following characters: _ . : / = + - @", s.UserStr(provided)),
```

Comment:
Do you think it would be easier to display the regex? Users can copy paste the regex into a regex matcher and test it out more easily.

Reply:
I think this is still clear enough since it's a list of allowed characters; I think it starts to get more confusing when order matters